CharlieBackend.Api.UnitTes/DashboardServiceTest.cs

```diff
@@ -1,22 +1,44 @@
 ﻿using System;
+using AutoMapper;
 using System.Collections.Generic;
 using System.Security.Claims;
 using System.Security.Principal;
 using System.Threading.Tasks;
 using CharlieBackend.Business.Services;
 using CharlieBackend.Core.DTO.Dashboard;
 using CharlieBackend.Core.Entities;
+using CharlieBackend.Core.Mapping;
 using CharlieBackend.Core.Models.ResultModel;
 using CharlieBackend.Data.Repositories.Impl.Interfaces;
 using Moq;
 using Xunit;
+using FluentAssertions;
 
 namespace CharlieBackend.Api.UnitTest
 {
     public class DashboardServiceTest : TestBase
     {
+        private readonly Mock<IDashboardRepository> _dashboardRepositoryMock;
+        private readonly IMapper _mapper;
+        private static long studentIdWithGroup = 7;
+        private static long studentIdWithoutGroup = 20;
+        private static long courseId = 1;
+        private static long courseIdWitoutGroup = 2;
+
+        public DashboardServiceTest()
+        {
+            _dashboardRepositoryMock = new Mock<IDashboardRepository>();
+            _mapper = GetMapper(new ModelMappingProfile());
+        }
+
+        private void InitializeForDashboardRepository()
+        {
+            _unitOfWorkMock.Setup(x => x.DashboardRepository).Returns(_dashboardRepositoryMock.Object);
+            _currentUserServiceMock = GetCurrentUserAsExistingStudent();
+        }
+
         [Fact]
-        public async Task GetStudentsClassbook()
+        public async Task GetStudentsClassbook_ExistingGroupIdWithExistingClassbook_ShouldReturnCurrentStudentsMarks()
```

Comment:
Let change ExistingGroupIdWithExistingClassbook to ValidDataPassed. As we will handle all invalid cases in the next tests.

Reply:
Updated like this https://github.com/ita-social-projects/WhatBackend/pull/409/commits/825d74fa93b7d2bd5ec07569c3b5ec3510e82d5c